### PR TITLE
Update shelly.markdown

### PR DESCRIPTION
**Pull request by ronschaeffer**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/shelly.markdown [hunk 1]
```diff
@@ -70,7 +70,33 @@ Shelly device relays are added to the Home Assistant by default as `switch` enti
...
 ## Events
 
-If the **BUTTON TYPE** of the switch connected to the device is set to `momentary` or `detached switch`, integration fires events when the switch is used. You can use these events in your automations.
+If the **BUTTON TYPE** of the switch connected to the device is set to `momentary` or `detached switch`, integration fires events under the type `shelly.click` when the switch is used. You can use these events in your automations.
+
+Also, some devices do not add an entity for the button/switch. For example, the Shelly Button1 has only one entity for the battery level. It does not have an entity for the button itself. To trigger automations based on button presses, use the `shelly.click` event.
+
+### Listening for events
+
+You can subscribe to the `shelly.click` event type in [Developer Tools/Events](https://www.home-assistant.io/docs/tools/dev-tools/) in order to examine the event data JSON for the correct parameters to use in your automations. For example, `shelly.click` returns event data JSON similar to the following when you press the Shelly Button1.
```
Comment (frenck) [inline source/_integrations/shelly.markdown]:
```suggestion
You can subscribe to the `shelly.click` event type in [Developer Tools/Events](/docs/tools/dev-tools/) in order to examine the event data JSON for the correct parameters to use in your automations. For example, `shelly.click` returns event data JSON similar to the following when you press the Shelly Button1.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Just wondering, why don't we point to device automation? Those are created to abstract this all away, right?

**Comment by ronschaeffer** [comment]
Good point. Is there a preferred way to show automations created with the UI in these docs? If I put in just the YAML, I think it could be confusing. The device automation uses the device ID e09c64a22553484d804353ef97a6fcd6 for my Shelly Button1, which is not at all obvious.

File: source/_integrations/shelly.markdown [hunk 2]
```diff
@@ -70,10 +70,42 @@ Shelly device relays are added to the Home Assistant by default as `switch` enti
...
 
-### Automation examples
+### Automations
+The simplest way to create automations is to use the Home Assistant automation editor. For example, to set an automation triggered by a double press of a particular Shelly Button1:
+1. In the Triggers section of the automation, set Trigger Type to `Device`.
+2. In the Device dropdown menu. find the Shelly Button1.
+3. In the Trigger dropdown menu, select `Button double clicked."
+4. Set any conditions and actions to complete your automation.
 
+You can also create automations using YAML, for example:
```
Comment (frenck) [inline source/_integrations/shelly.markdown]:
```suggestion
### Automations

The simplest way to create automations is to use the Home Assistant automation editor. For example, to set an automation triggered by a double press of a particular Shelly Button1:

1. In the Triggers section of the automation, set Trigger Type to `Device`.
2. In the Device dropdown menu. find the Shelly Button1.
3. In the Trigger dropdown menu, select `Button double clicked`.
4. Set any conditions and actions to complete your automation.

You can also create automations using YAML, for example:

```

**Review by frenck — APPROVED** [review]
Nice, @ronschaeffer! 👍 

Well done!

../Frenck

**Pull request merged**